client/src/app/components/page/task-detail/task-detail.component.html

```diff
@@ -1,27 +1,35 @@
-<div class="card mt-4 container flex-row">
+<div class="mt-4 container border rounded-lg">
     <div class="row">
         <div class="colCard col-sm">
             <form [formGroup]="taskForm" (submit)="onSubmit()">
```

Comment:
You can wrap the whole form withing bootstrap "Card" https://getbootstrap.com/docs/4.5/components/card/ this way you will get proper spacing and borders. Please do so.

Reply:
Done